### ISSUE TITLE
Failed to look up view

### ISSUE THREAD

**Issue by sainthasras** [deriegle/dart-express#6]
Hello, I tried creating a new project with  dart-express. Anytime I try to access a page with jael or mustache template, I keep getting this error:
"Failed to lookup view \"index.mustache\" in views directory \"/views\""

Please what is the problem?

**Comment by deriegle** [comment]
Hey there, I'd love to help you get this set up and working.

Could you post a sample of the code you have and how your folder structure is set up?

You should be able to put a `index.mustache` file in a `views` folder with this code and it will work.

```dart
import 'dart:io';
import 'package:dart_express/dart_express.dart';

const int PORT = 5000;

main() {
  var app = express();

  app.engine(MustacheEngine.use());
  app.settings.viewEngine = 'mustache';

  app.get('/', (req, res) {
    res.render('index');
  });

  app.listen(PORT, (int port) => print('Listening on port $port'));
}
```

**Comment by sainthasras** [comment]
I don't know what the problem is but I seem not to get this working. The page is still not loaded. I placed index.mustache file in '**root project/views/index.mustache'**

**Comment by deriegle** [comment]
Hey there @sainthasras,
I just published version `0.2.2` of the `dart-express` library that fixes the issue you were seeing related to views. I've also fixed how it searches for views to make it smarter.

By default it will look for `views` in the root of the directory. If you'd like to set a custom directory you can do so very easily using this command:

```dart
app.set('views', './lib/views');
```

```dart
import 'package:dart_express/dart_express.dart';

void main() {
  var app = express();

  app.engine(MustacheEngine.use());
  app.set('view engine', 'mustache');

  app.get('/', (req, res) {
    res.render('index');
  });

  app.listen(3001, (int port) => print('Listening on $port'));
}
```

**Comment by sainthasras** [comment]
Right. Thanks a lot.
